plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/validator/NumericValidator.java

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.openshift.internal.ui.validator;
+
```

Comment:
any particular reason for *NOT* putting this into package *org.jboss.tools.openshift.internal.common.ui.databinding*?

Reply:
Done with commit 6462280